clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -3161,7 +3164,8 @@ check_kernel_syscall_trace(void)
     }
     {
         std::vector<memref_with_IR_t> memref_setup = {
-            { gen_marker(TID_A, TRACE_MARKER_TYPE_FILETYPE, FILE_TYPE), nullptr },
+            { gen_marker(TID_A, TRACE_MARKER_TYPE_FILETYPE, FILE_TYPE_FULL_SYSCALL_TRACE),
```

Comment:
Seems worth a comment that A) we want this flag to trigger extra invariant checks and B) this is ok b/c there are no loads/stores in the trace.

DItto below.

Reply:
Done.